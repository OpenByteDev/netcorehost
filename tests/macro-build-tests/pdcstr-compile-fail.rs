fn main() {
    // with internal nul 
    netcorehost::pdcstr!("\0");
    netcorehost::pdcstr!("somerandomteststring\0");
    netcorehost::pdcstr!("somerandomteststring\0somerandomteststring");
    netcorehost::pdcstr!("somerandomteststring\0somerandomteststring");
}
