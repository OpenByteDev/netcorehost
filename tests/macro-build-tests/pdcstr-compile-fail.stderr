error: nul byte found in the literal
 --> $DIR/pdcstr-compile-fail.rs:3:26
  |
3 |     netcorehost::pdcstr!("\0");
  |                          ^^^^

error: nul byte found in the literal
 --> $DIR/pdcstr-compile-fail.rs:4:26
  |
4 |     netcorehost::pdcstr!("somerandomteststring\0");
  |                          ^^^^^^^^^^^^^^^^^^^^^^^^

error: nul byte found in the literal
 --> $DIR/pdcstr-compile-fail.rs:5:26
  |
5 |     netcorehost::pdcstr!("somerandomteststring\0somerandomteststring");
  |                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: nul byte found in the literal
 --> $DIR/pdcstr-compile-fail.rs:6:26
  |
6 |     netcorehost::pdcstr!("somerandomteststring\0somerandomteststring");
  |                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
